sycl/test-e2e/Graph/Inputs/host_task.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "../graph_common.hpp"
 
+#include <sycl/detail/host_task_impl.hpp>
+
```

Comment:
Why are we including detail headers here? Are users expected to do the same?

Reply:
Thanks for the context, I do agree with some of the concerns around presenting this to users but if it's planned to document this appropriately then that's fine.